products/meilisearch.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Meilisearch
+category: server-app
+tags: meilisearch
+iconSlug: meilisearch
+permalink: /meilisearch
+versionCommand: meilisearch --version
+releasePolicyLink: https://www.meilisearch.com/docs/learn/update_and_migration/versioning
+changelogTemplate: https://github.com/meilisearch/meilisearch/releases
+releaseDateColumn: true
+
+identifiers:
+-   repology: meilisearch
+
+auto:
+-   git: https://github.com/meilisearch/meilisearch.git
+
+releases:
+-   releaseCycle: "1.5"
+    releaseDate: 2023-11-20
+    eol: false
+    latest: "1.5.1"
+    latestReleaseDate: 2023-12-13
+
+---
+
+> [Meilisearch](https://www.meilisearch.com/) is a RESTful search API. It aims to be a ready-to-go solution for everyone who wants a fast and relevant search experience for their end-users.
+
+Meilisearch releases and their associated binaries are available [in this GitHub page](https://github.com/meilisearch/meilisearch/releases).
+
+The binaries are versioned following [SemVer conventions](https://semver.org/). To know more, read our [versioning policy](https://github.com/meilisearch/engine-team/blob/main/resources/versioning-policy.md).
```

Comment:
```suggestion
Meilisearch [follows semver](https://github.com/meilisearch/engine-team/blob/main/resources/versioning-policy.md).
Minor versions release cadence is between four and six times a year.
Only the the latest engine release is maintained with high-priority bug and security fixes.
```

This description is missing the release cadence and support policy.
The suggestion is based on what I read on https://www.meilisearch.com/docs/learn/update_and_migration/versioning.